examples/vm_scheduling/reinforcement_learning/ac/agent/models/combine_net.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
for the codes shared by `ac` and `dqn`, you can put the codes under `examples/vm_scheduling/reinforcement_learning/...` to reuse them, instead of copying and pasting